isofit/test/test_wray.py

```diff
@@ -24,8 +24,6 @@ def test_decorators():
     jobs = [decorator.remote(a, b) for a, b in cases.values()]
     assert ray.get(jobs) == list(cases.keys())
 
-    return True
```

Comment:
These `return True` trigger deprecated `pytest` behavior. They do not actually leverage this behavior, and can safely be removed.